src/token-group/token.tsx

```diff
@@ -21,10 +20,7 @@ interface TokenProps extends ItemAttributes {
 
 export function Token({ disabled, dismissLabel, onDismiss, children }: TokenProps) {
   return (
-    <div
-      className={clsx(styles.token, disabled && styles['token-disabled'])}
-      aria-disabled={disabled ? 'true' : undefined}
```

Comment:
The Token is a visual but not semantic container. When rendered with TokenList the aria-disabled is set on list elements. When rendered as standalone the aria-disabled must be provided separately:

```
<div role="group" aria-label="Token" aria-disabled={true}>
  <Token>Token</Token>
<div>
```